### PR TITLE
Feature: Add reset and verification screens

### PR DESCRIPTION
**Pull request by VladyslavP**

### CODE REVIEW

File: src/mobile/components/Modal/Modal.tsx [hunk 1]
```diff
@@ -44,8 +44,12 @@ const Modal = props => {
...
                 <div className="cr-mobile-modal__header">
                     <div className="cr-mobile-modal__header-back" onClick={handleOnBack}>
-                        <ArrowIcon/>
-                        <span>{props.backTitle}</span>
+                        {props.backTitle &&
+                        <React.Fragment>
+                          <ArrowIcon/>
+                          <span>{props.backTitle}</span>
+                        </React.Fragment>
+                        }
```
Comment (oyershov) [inline src/mobile/components/Modal/Modal.tsx:52]:
```suggestion
                        {props.backTitle && (
                            <React.Fragment>
                              <ArrowIcon/>
                              <span>{props.backTitle}</span>
                            </React.Fragment>
                        )}
```

File: src/screens/EmailVerification/index.tsx [hunk 2]
```diff
@@ -53,12 +54,30 @@ class EmailVerificationComponent extends React.Component<Props> {
...
                         <div className="pg-emailverification-body-text">{text}</div>
+                        {
+                            !isMobileDevice &&
+                            <div className="pg-emailverification-body-container">
+                                {emailVerificationLoading ? <Spinner animation="border" variant="primary"/> :
+                                    <button className="pg-emailverification-body-container-button"
+                                            onClick={this.handleClick}>{button}</button>}
+                            </div>
+                        }
+                        {isMobileDevice &&
```
Comment (oyershov) [inline src/screens/EmailVerification/index.tsx:68]:
```suggestion
                        {isMobileDevice && (
```

File: src/screens/EmailVerification/index.tsx [hunk 3]
```diff
@@ -53,12 +54,30 @@ class EmailVerificationComponent extends React.Component<Props> {
         return (
             <div className="pg-emailverification-container">
                 <div className="pg-emailverification">
-                    <div className="pg-emailverification-title">{title}</div>
+                    {!isMobileDevice && <div className="pg-emailverification-title">{title}</div>}
                     <div className="pg-emailverification-body">
                         <div className="pg-emailverification-body-text">{text}</div>
+                        {
+                            !isMobileDevice &&
```
Comment (oyershov) [inline src/screens/EmailVerification/index.tsx:61]:
```suggestion
                        {!isMobileDevice && (
```

File: src/screens/EmailVerification/index.tsx (hunk 2 above)
Comment (VladyslavP) [reply src/screens/EmailVerification/index.tsx:68]: this is something TSLint should be responsible for

**Pull request merged**